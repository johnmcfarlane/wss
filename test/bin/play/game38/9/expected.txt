 48 10,6  | FOGEY
 45  9,7  | FODGE
 45  9,7  | HEDGY
 44 10,6  | GEHEY
 41  6,11 - GOOFY
 41  6,11 - HOOFY
 39  9,7  | HODGE
 38  2,6  | FEH
 38  2,6  | FOH
 36  8,8  - FEH
 36  8,8  - FEY
 36  8,8  - FOH
 36  8,8  - FOY
 36 10,6  | GOOEY
 36 10,6  | HOOEY
 36 10,6  | YOGEE
 34  2,6  | YEH
 33  2,6  | FG
 33  8,10 - OOFY
 32  8,10 - FEH
 32  8,8  - FOE
 32  8,10 - FOH
 32  8,8  - FOO
 32  6,11 - GOOEY
 32  6,11 - GOOF
 32  2,6  | HF
 32  6,11 - HOOEY
 32  6,11 - HOOF
 32  6,11 - HOOYE
 31  8,8  - YOHO
 30  8,8  - GEY
 30  8,8  - GOOF
 30  8,8  - GOY
 30  8,8  - GYE
 30  8,8  - HEY
 30  8,8  - HOOF
 30  8,8  - HOY
 30  8,8  - HYE
 30  8,8  - YEH
 29  8,8  - FE
 29 10,3  - FG
 29  2,6  | HG
 29  8,8  - HOEY
 29  6,11 - HOEY
 29  6,11 - HOGO
 28 10,3  - HF
 27  9,8  | EDGY
 27  2,6  | FE
 27  8,10 - OOF
 26  8,8  - GEO
 26  8,8  - GOO
 26  8,8  - HEO
 26  8,8  - HOE
 26  8,8  - HOO
 25 10,3  - HG
 25  9,10 - OFO
 24 10,2  | CHEGO
 24 10,2  | COHOG
 24  2,6  | EF
 24 10,10 - FG
 24  8,10 - FOE
 24  8,10 - FOO
 23  9,10 - EF
 23 10,3  - FE
 23  2,6  | GO
 23  8,8  - GO
 23  2,6  | HE
 23  8,8  - HE
 23  2,6  | HO
 23  8,8  - HO
 23  9,10 - OF
 23  8,8  - OOFY
 23  8,10 - OOH
 23  2,6  | YE
 23  8,8  - YE
 23  9,10 - YEH
 23  2,6  | YO
 23  8,8  - YO
 22 10,10 - HF
 21  9,9  | DEF
 21  2,6  | EH
 21  9,10 - OHO
 20 10,2  | COOEY
 20 10,10 - FE
 20 10,10 - HG
 20  8,8  - OOF
 19  9,10 - EH
 19  9,10 - EOF
 19 10,3  - GO
 19 10,3  - HE
 19 10,3  - HO
 19  9,10 - OH
 19  9,10 - OOF
 18  9,9  | DEG
 18  9,9  | DEY
 18  9,9  | DOG
 18  9,9  | DOH
 18  9,9  | DYE
 18 10,6  | FOGE
 18 10,5  | HOOYE
 18  8,8  - OOH
 18  6,6  - SEHYO
 18 10,5  | YOGEE
 17  8,8  | FIGO
 17  9,4  | GOOFED
 17  9,4  | HOOFED
 17  9,10 - OOH
 17  7,6  | OYEZ
 17  9,10 - YE
 17  9,10 - YO
 16 10,8  | FEH
 16 10,8  | FEY
 16  8,8  | FIG
 16  8,6  | FOGIE
 16 10,6  | GHEE
 16 10,7  | GHEE
 16 10,10 - HE
 16 10,10 - HO
 16 10,7  | HOEY
 16  8,8  | YIGH
 15  2,6  | EO
 15  7,7  | FEZ
 15  8,8  - OY
 14 10,10 - EF
 14  1,6  - EYEN
 14  2,6  - FEN
 14  8,8  | FIE
 14 10,8  | GEY
 14  7,7  | GEZ
 14  8,8  | GIF
 14  1,6  - GOEN
 14 10,8  | HEY
 14  5,11 - HG
 14 10,10 - OF
 14  9,8  - OFO
 14  5,11 - OHOY
 14  6,6  - SOOEY
 14  9,8  - YEH
 14 10,8  | YEH
 14  7,7  | YEZ
 14  8,6  | YOGI
 13  9,8  - EF
 13  8,10 - FE
 13  8,8  | FI
 13  8,7  | GHI
 13  5,11 - HEY
 13  9,8  - OF
 13  3,8  - SIGH
 12 10,2  | CHEF
 12  9,9  | DOE
 12  9,9  | DOO
 12  2,7  | EH
 12 10,10 - EH
 12 10,9  | EOF
 12 10,8  | FEE
 12 10,8  - FG
 12  9,6  | FOODY
 12  2,6  - GEN
 12  6,11 - GEY
 12  6,11 - GOY
 12  6,11 - GYE
 12  2,6  - HEN
 12  6,11 - HEY
 12  6,11 - HF
 12  6,11 - HOG
 12  6,11 - HOY
 12  9,7  | HYDE
 12  6,11 - HYE
 12 10,6  | OGEE
 12 10,7  | OGEE
 12  2,7  | OH
 12 10,10 - OH
 12  8,7  | OHIO
 12  9,8  - OHO
 12  9,5  | OOHED
 12 10,10 - OY
 12  3,5  - PEENGE
 12  2,6  - YEN
 12  9,7  | YODH
 11 10,2  | CFH
 11 10,2  | CHOY
 11  5,11 - EGO
 11  9,8  - EH
 11  9,10 - EO
 11  9,8  - EOF
 11  9,7  | FED
 11  9,7  | FOD
 11  8,8  | GIE
 11  8,8  | GIO
 11  9,6  | GOODY
 11  5,11 - HEO
 11  7,6  | HF
 11 10,8  - HF
 11  6,11 - HG
 11  8,8  | HIE
 11  9,6  | HOODY
 11  9,7  | HYD
 11  8,8  - OE
 11  9,10 - OE
 11  9,8  - OH
 11  5,11 - OHO
 11  8,8  - OO
 11  9,10 - OO
 11  9,8  - OOF
 11  9,8  - OY
 11  8,10 - OY
 11  9,8  - YE
 11  9,8  - YO
 10 10,2  | CHG
 10 10,2  | COEF
 10 10,2  | COOF
 10 10,9  | EF
 10 10,9  | EGO
 10  5,11 - EH
 10  8,8  | EIGH
 10 10,9  | EYE
 10 10,8  - FE
 10  1,7  - FEATY
 10 10,8  | GEE
 10 10,8  | GEO
 10  6,11 - GEO
 10  8,8  | GI
 10  9,7  | GODE
 10  6,11 - GOO
 10  5,11 - HE
 10 10,8  | HEE
 10 10,8  | HEO
 10  6,11 - HEO
 10  7,6  | HG
 10 10,8  - HG
 10  8,8  | HI
 10  6,11 - HOE
 10  6,11 - HOO
 10  5,11 - OH
 10  9,6  | OHED
 10  9,8  - OOH
 10 10,8  | YEE
 10  8,8  | YI
 10  9,7  | YODE
  9 10,2  | COHO
  9 10,2  | COYO
  9  4,4  - DALF
  9  6,11 - EOF
  9  6,8  | FASH
  9  2,7  - FATH
  9  9,6  | FEOD
  9  9,6  | FOOD
  9  9,7  | GED
  9  6,11 - GO
  9  1,7  - GOATY
  9  9,7  | GOD
  9  6,11 - HE
  9  9,7  | HED
  9  6,11 - HO
  9  9,7  | HOD
  9  9,7  | YOD
  8 10,2  | CF
  8 10,2  | CHE
  8 10,2  | COG
  8 10,2  | COY
  8  2,6  - EEN
  8  6,11 - EF
  8  6,11 - EGO
  8 10,9  | EH
  8 10,10 - EO
  8  8,7  | FEI
  8  1,8  - FOSIE
  8  6,8  | GASH
  8  9,6  | GEOD
  8  1,7  - GHAT
  8  9,6  | GOOD
  8  7,6  | HE
  8 10,8  - HE
  8  7,6  | HO
  8 10,8  - HO
  8  9,6  | HOED
  8  9,6  | HOOD
  8 10,10 - OE
  8 10,10 - OO
  8  6,6  - SHOG
  8 10,8  - YE
  8 10,8  - YO
  7 10,2  | CG
  7 10,2  | CH
  7 10,8  - EF
  7  6,11 - EH
  7  9,8  - EO
  7  2,7  - FATE
  7  1,7  - FEAT
  7  8,7  | GOI
  7 10,7  | GYE
  7  8,7  | HEI
  7  8,7  | HOI
  7 10,7  | HYE
  7  9,8  - OE
  7  8,10 - OE
  7  7,6  | OF
  7 10,8  - OF
  7  9,8  - OO
  7  8,10 - OO
  7  6,6  - SHY
  6 10,2  | COE
  6 10,2  | COO
  6  4,4  - DALE
  6  6,8  | EASY
  6  2,7  - EATH
  6  9,8  | EDH
  6 10,8  - EH
  6  6,8  | FAS
  6  2,7  - FAT
  6 10,7  | FEE
  6 10,7  | FOE
  6  2,9  | FOR
  6  2,7  - GATE
  6  1,7  - GEAT
  6  1,7  - GOAT
  6  2,7  - HATE
  6  1,7  - HEAT
  6  8,9  | IFE
  6  8,9  | IOF
  6  2,7  - OATH
  6  2,7  - OATY
  6  5,11 - OE
  6  7,6  | OH
  6 10,8  - OH
  6  7,6  | OY
  6 10,8  - OY
  6  6,6  - SEGO
  6  6,6  - SHOE
  6  6,6  - SHOO
  6  6,6  - SOHO
  6  2,7  - YATE
  6  1,7  - YEAT
  5  6,9  | ASG
  5  6,9  | ASH
  5  9,9  | DY
  5  6,11 - EO
  5  8,1  | ETH
  5  8,1  | ETY
  5 10,7  | EYE
  5 10,8  | FE
  5  6,8  | GAS
  5  2,7  - GAT
  5 10,7  | GEE
  5  2,9  | GOR
  5  6,8  | HAS
  5  2,7  - HAT
  5 10,7  | HEE
  5 10,7  | HOE
  5  2,9  | HOR
  5  2,8  - HSI
  5  8,9  | IF
  5  6,6  - SEG
  5  6,6  - SHE
  5  6,6  - SHO
  5  3,8  - SIG
  5  6,6  - SOG
  5  6,6  - SOH
  5  6,6  - SOY
  5  6,6  - SYE
  5  6,8  | YAS
  5  2,7  - YAT
  5  9,8  | YD
  5 10,7  | YEE
  5  2,9  | YOR
  4  9,8  | EDO
  4 10,9  | EE
  4 10,8  - EO
  4 10,9  | EO
  4 10,8  | HE
  4  9,8  | ODE
  4  7,6  | OE
  4 10,8  - OE
  4  8,8  | OI
  4  7,6  | OO
  4 10,8  - OO
  4  6,6  - SH
  4 10,8  | YE
  3  6,9  | ASE
  3  3,7  - ATE
  3  9,9  | DE
  3  9,9  | DO
  3  2,7  - EAT
  3  9,8  | ED
  3  2,7  - OAT
  3  9,8  | OD
  3  3,8  - SIE
  3  6,6  - SOE
  3  5,10 | tH
  2 10,8  | EE
  2  8,9  | IO
  2 10,8  | OE
  2  6,6  - SO
  1  5,10 | tE
  1  5,10 | tO
