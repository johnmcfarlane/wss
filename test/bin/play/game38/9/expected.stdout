 48 10,6  | FOGEY
 45  9,7  | HEDGY
 36 10,6  | GOOEY
 36 10,6  | HOOEY
 36 10,6  | YOGEE
 32  8,10 - FEH
 32  8,10 - FOH
 32  6,11 - HOOEY
 32  6,11 - HOOF
 30  8,8  - GEY
 30  8,8  - GOOF
 30  8,8  - GOY
 30  8,8  - HEY
 30  8,8  - HOOF
 30  8,8  - HOY
 27  9,8  | EDGY
 27  2,6  | FE
 26  8,8  - GOO
 24 10,2  | COHOG
 23  9,10 - EF
 23  2,6  | GO
 23  8,8  - GO
 23  2,6  | HE
 23  8,8  - HE
 23  2,6  | HO
 23  8,8  - HO
 23  9,10 - OF
 23  2,6  | YE
 23  2,6  | YO
 21  9,9  | DEF
 21  9,10 - OHO
 20 10,2  | COOEY
 20 10,10 - FE
 19  9,10 - EH
 19  9,10 - OH
 18  9,9  | DEY
 18  9,9  | DOG
 18  9,9  | DYE
 18  8,8  - OOH
 18 10,5  | YOGEE
 17  9,4  | GOOFED
 17  9,4  | HOOFED
 17  7,6  | OYEZ
 16 10,8  | FEH
 16 10,8  | FEY
 16  8,8  | FIG
 16  8,6  | FOGIE
 16 10,6  | GHEE
 16 10,7  | GHEE
 16 10,10 - HE
 16 10,10 - HO
 15  7,7  | FEZ
 14  1,6  - EYEN
 14  2,6  - FEN
 14  8,8  | FIE
 14 10,8  | GEY
 14 10,8  | HEY
 14  6,6  - SOOEY
 14 10,8  | YEH
 14  8,6  | YOGI
 13  9,8  - EF
 13  8,10 - FE
 13  8,7  | GHI
 13  9,8  - OF
 13  3,8  - SIGH
 12 10,2  | CHEF
 12  9,9  | DOE
 12 10,8  | FEE
 12  2,6  - GEN
 12  2,6  - HEN
 12  6,11 - HEY
 12  6,11 - HOG
 12  6,11 - HOY
 12 10,6  | OGEE
 12 10,7  | OGEE
 12  9,8  - OHO
 12  9,5  | OOHED
 12  2,6  - YEN
 12  9,7  | YODH
 11  9,8  - EH
 11  9,7  | FED
 11  8,8  | GIE
 11  9,6  | GOODY
 11  8,8  | HIE
 11  9,6  | HOODY
 11  8,8  - OE
 11  9,8  - OH
 11  5,11 - OHO
 11  9,8  - OY
 10 10,2  | COOF
 10 10,9  | EF
 10 10,9  | EGO
 10  5,11 - EH
 10 10,9  | EYE
 10 10,8  - FE
 10 10,8  | GEE
 10  8,8  | GI
 10  8,8  | HI
 10  6,11 - HOE
 10  5,11 - OH
 10  9,6  | OHED
 10  9,8  - OOH
  9 10,2  | COHO
  9  6,8  | FASH
  9  9,6  | FEOD
  9  9,6  | FOOD
  9  9,7  | GED
  9  9,7  | GOD
  9  6,11 - HE
  9  6,11 - HO
  9  9,7  | HOD
  9  9,7  | YOD
  8 10,2  | COG
  8 10,2  | COY
  8 10,9  | EH
  8  6,8  | GASH
  8  1,7  - GHAT
  8  9,6  | GOOD
  8  7,6  | HE
  8 10,8  - HE
  8  7,6  | HO
  8 10,8  - HO
  8  9,6  | HOED
  8  9,6  | HOOD
  8  6,6  - SHOG
  8 10,8  - YE
  8 10,8  - YO
  7  2,7  - FATE
  7  1,7  - FEAT
  7  7,6  | OF
  7 10,8  - OF
  7  6,6  - SHY
  6 10,2  | COO
  6  4,4  - DALE
  6  6,8  | EASY
  6  2,7  - EATH
  6  9,8  | EDH
  6  6,8  | FAS
  6  2,7  - FAT
  6 10,7  | FEE
  6 10,7  | FOE
  6  2,9  | FOR
  6  2,7  - GATE
  6  1,7  - GOAT
  6  2,7  - HATE
  6  1,7  - HEAT
  6  2,7  - OATH
  6  7,6  | OH
  6 10,8  - OH
  6  7,6  | OY
  6 10,8  - OY
  6  6,6  - SEGO
  6  6,6  - SHOE
  6  6,6  - SHOO
  5  6,9  | ASH
  5  8,1  | ETH
  5 10,7  | EYE
  5 10,8  | FE
  5  6,8  | GAS
  5  2,7  - GAT
  5 10,7  | GEE
  5  2,9  | GOR
  5  6,8  | HAS
  5  2,7  - HAT
  5 10,7  | HOE
  5  8,9  | IF
  5  6,6  - SEG
  5  6,6  - SHE
  5  6,6  - SOY
  4 10,8  | HE
  4  9,8  | ODE
  4  7,6  | OE
  4 10,8  - OE
  4  8,8  | OI
  4  6,6  - SH
  4 10,8  | YE
  3  3,7  - ATE
  3  9,9  | DE
  3  9,9  | DO
  3  2,7  - EAT
  3  9,8  | ED
  3  2,7  - OAT
  3  9,8  | OD
  2 10,8  | OE
  2  6,6  - SO
  1  5,10 | tE
  1  5,10 | tO
